diff --git a/src/python/truchas/TruchasEnvironment.py b/src/python/truchas/TruchasEnvironment.py
index 8d4e2e90..626f53a1 100644
--- a/src/python/truchas/TruchasEnvironment.py
+++ b/src/python/truchas/TruchasEnvironment.py
@@ -180,6 +180,7 @@ class TruchasEnvironment:
         :return: stdout and Truchas output data.
         :rtype: str, TruchasData
         """
+        nprocs = 1 # TEMPORARY OVERRIDE FOR PUBLIC SERVER TESTING
 
         # find the absolute path to the input file
         input_name = os.path.splitext(input_file)[0]
